app/controllers/api/versions_controller.rb

```diff
@@ -0,0 +1,8 @@
+class API::VersionsController < API::RestfulController
+
+  private
+
+  def accessible_records
+    load_and_authorize(params[:model]).versions.where(event: 'update')
```

Comment:
Let's order this by `(created_at: :desc)` so we can ensure we always get the latest versions?